packages/fxa-auth-server/scripts/convert-customers-to-stripe-automatic-tax.ts

```diff
@@ -0,0 +1,328 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import program from 'commander';
+import Stripe from 'stripe';
+import Container from 'typedi';
+
+import { setupProcessingTaskObjects } from '../lib/payments/processing-tasks-setup';
+import { AppConfig, AuthFirestore } from '../lib/types';
+
+const pckg = require('../package.json');
+
+const MONTHLY_NOTICE_DAYS = 14;
+const YEARLY_NOTICE_DAYS = 30;
```

Comment:
Q: Is this name semi-misleading if _"**6 month** and yearly plans require at least 30 days of notice"_?

Reply:
Fair point - hard to think of a good name 🤔